CONTRIBUTING.md

```diff
@@ -1,3 +1,22 @@
 # Contributing to Screwdriver
 
 Have a look at our guidelines, as well as pointers on where to start making changes, in our official [documentation](http://docs.screwdriver.cd/about/contributing).
+
+## Commit message format
+
+We use [semantic-release](https://www.npmjs.com/package/semantic-release), which requires commit messages to be in this specific format: `<type>(<scope>): <subject>`
```

Comment:
we don't need to repeat the instructions here https://docs.screwdriver.cd/about/contributing/#contributing is our source of truth.

Reply:
Removed